diff --git a/libzmq.rb b/libzmq.rb
index 5d50af6..3533377 100755
--- a/libzmq.rb
+++ b/libzmq.rb
@@ -21,10 +21,10 @@ LIBNAME="libzmq.a"
 ROOTDIR=File.absolute_path(File.dirname(__FILE__))
 LIBSODIUM_DIST=File.join(ROOTDIR, "dist")
 VALID_ARHS_PER_PLATFORM = {
-  "iOS"     => ["armv7", "armv7s", "arm64", "i386", "x86_64"],
-  "macOS"   => ["x86_64"],
-  "tvOS"    => ["arm64", "x86_64"],
-  "watchOS" => ["armv7k", "i386"],
+  "iOS"     => ["arm64", "x86_64"],
+  "macOS"   => [],
+  "tvOS"    => [],
+  "watchOS" => [],
 }
 
 DEVELOPER               = `xcode-select -print-path`.chomp
@@ -255,6 +255,8 @@ for platform in PLATFORMS
       "--enable-static",
       "--host=#{host}",
       "--with-libsodium=#{LIBSODIUM_DIST}/#{platform}",
+      "--disable-perf",
+      "--disable-curve-keygen",
     ]
     exit 1 unless system(configure_cmd.join(" "))
 
